rtems/RTEMSThread.java

```diff
@@ -15,6 +15,8 @@ public RTEMSThread(int priority, Thread.State state){
 		this();
 		this.currentPriority = this.realPriority = priority;
 		this.state = state;
+		//this.lockMutex = new ChainControl();
+		PriorityQueue<RTEMSThread> wait = null;
```

Comment:
I have made changes as per your suggestion. Please have a look at it in updated code.